src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -113,7 +120,19 @@ private function execute(array $events, string $payload, Document $webhook, Docu
         }
 
         if (false === \curl_exec($ch)) {
-            $this->errors[] = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
+            $errorCount = $webhook->getAttribute('errors', 0) + 1;
+            $lastErrorLogs = \curl_error($ch) . ' in events ' . implode(', ', $events) . ' for webhook ' . $webhook->getAttribute('name');
```

Comment:
Webhook name isnt valuable here, as it's stored alongside all other webhook info on it's document.

Alongside curl_error I would also try to store HTTP status code and response body. Body should be truncated to ensure long ones don't cause updateDocument to fail.

Reply:
When I'm adding `$statusCode = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);`, the status code is `200` even when exceptions are thrown. Any idea why that might be happening?

Also, how to fetch response body? In this case `\curl_exec($ch)` is `false`.

Reply:
When I'm adding `$statusCode = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);`, the status code is `200` even when exceptions are thrown. Any idea why that might be happening?

Also, how to fetch response body? In this case `\curl_exec($ch)` is `false`.